### DISCUSSION TITLE
jmbuhr.de - Why the Dunning-Krueger Effect is Probably Not Real

### DISCUSSION THREAD

**Discussion by giscus** [jmbuhr/jmbuhr.github.io#5] [General]
# jmbuhr.de - Why the Dunning-Krueger Effect is Probably Not Real

https://jmbuhr.de/posts/2021-03-20-why-the-dunning-krueger-effect-is-probably-not-real/index.html

**Comment by enuhfer** [comment]
Hi! Nice post, and thank you for creating it.  I do like the examples you provided. 

Your middle citation above from Numeracy, also noted by Jarry, did in fact provide details of the "somewhat statistical explanation of why this happens." That paper is actually a continuation of Nuhfer et al. 2016 http://dx.doi.org/10.5038/1936-4660.9.1.4. Previous scholars noted the problem of regression to the mean, but the two Numeracy papers did something a bit different. They compiled an actual dataset of 1154 participants (something one cannot generate on a computer), simulated a dataset devoid of human response (computerized random responses to the instruments used to obtain the paired measures) then compared both through the Dunning-Kruger convention and alternatives to using that convention. The examples are in both papers and in Appendices. 

The raw data and the random numbers used are in Excel files the appendices, so you can provide your students an actual dataset to allow them to grapple with this problem. An outcome is being able to understand what randomness looks like in varied graphical presentations. What randomness is going to look like is usually mis-imagined. Simulating actual data with random numbers allows seeing this, but you have to do the simulations of students responses on the instruments, and not trust just generating random scores bounded by 0 and 100 to give a clear picture.

The team published a social justice application paper in Numeracy in 2019 https://doi.org/10.5038/1936-4660.12.2.2  and shared an even larger dataset in that Appendix that can be used for teaching the same.

Your note that "...it feels so right that the skills required for an accurate self-assessment would be linked to the skills required for the actual task" is what made the Kruger-Dunning graph so seductive. Researchers, reviewers, and editors alike produced peer-reviewed papers for over twenty years without realizing that their "replicable results" were being produced by the graphic and not by human self-assessments.

See presentations at the National Numeracy Network annual meetings https://www.youtube.com/watch?v=oRuGd9I-Hzw (2021)
and  https://www.youtube.com/watch?v=5P4CldES5RQ (2022)  for the importance of self-assessment in becoming educated. Mistaking most people as "unskilled and unaware of it"  is a disservice. Psychologists are starting to realize just how great that disservice is (https://doi.org/10.1098/rsos.191727 and https://doi.org/10.1111/bjop.12553).

Best wishes, Ed

**Reply by jmbuhr** [reply]
Thank you for the additional references, very insightful!
